src/landingPage.scss

```diff
@@ -295,33 +370,36 @@
     }
 
     .right-on-video-container {
-        width: 100%;
-        height: 100%;
+        position: relative;
+        padding-bottom: 56.25%; /* 16:9 aspect ratio */
```

Comment:
would you mind giving me a bit more info on the `padding-bottom` property here? I'm wondering how the padding itself impacts the layout and how you arrived at 56.25%

Reply:
Looks like we're using SASS, so this value can be extracted into a variable and used across the file instead of duplicating the value

Reply:
I can explain the padding-bottom value. When I switched the video it would get these white borders on the sides, I'm assuming because of the aspect ratio not matching the container, so I ask ChatGPT and it give my this bit of code to fix the issue and that value is based on the aspect ratio of the video. 9/16

Reply:
huh, learned something new every day. Apparently, `iframes` don't work like images in terms of scaling. Maybe it's just me but I think including this link: http://alistapart.com/article/creating-intrinsic-ratios-for-video/ would help as a comment (instead of 16:9 aspect ratio) so that other people who see this know where the `padding-bottom` is coming from. 

And Mani's point is a good one. you can define a variable in the scss file like: `$video-aspect-ratio: 56.25%;` (underneath the imports in the file) and then use it like `padding-bottom: $video-aspect-ratio` (in `.right-on-video-container`). See this for reference: https://sass-lang.com/documentation/variables and shoot me a message on slack if you need a hand

Reply:
Gotcha I did add the variable and placed the link in the file since it does a better job of explaining how that works!

Reply:
<img width="100" alt="Screenshot 2023-05-05 at 1 45 50 PM" src="https://user-images.githubusercontent.com/102615547/236533828-0b4455bc-bb05-443f-af7e-eb0589f0ffab.png">
<img width="100" alt="Screenshot 2023-05-05 at 1 54 59 PM" src="https://user-images.githubusercontent.com/102615547/236533889-d8239e72-22f8-4081-87d8-bf4bfd6825d2.png">

Also Drew here's some screenshots of what problems i was getting with the old code and why I used the padding-bottom tag

Reply:
totally, I did some testing and got the exact same result but didn't know about the padding-bottom trick. Good to know! I think adding the url will help clarify this if it isn't common knowledge. nice job